doc/sphinx/source/shared/mxnet_tutorial.rst

```diff
@@ -0,0 +1,101 @@
+.. shared/mxnet_tutorial.rst:
+
+
+Working with subgraphs on MXNet
+################################
+
+One DL framework with advancing efforts on shared graph optimizations is Apache 
+MXNet\*, where Intel has contributed efforts showing how to work with our
+nGraph Compiler stack as an `experimental backend`_. Our approach provides 
+**more opportunities** to start working with different kinds of graph 
+optimizations **than would be available to the MXNet framework alone**, for 
+reasons outlined in our `features`_ documentation.
+
+.. TODO : Link to latest on mxnet when/if they do this instead of linking to PR;
+   keep in mind this tutorial will still work  regardless of the merge status of 
+   the experimental backend if you already use the ngraph-mxnet Github repo 
+
+The Intel nGraph Compiler provides an in-memory abstraction layer for converting 
+the mathematical representation of a DL model into an optimized-for-execution 
+format that can be understood *by* and run *on* multiple hardware backends. A 
+primary goal of this integration is to provide a seamless development and 
+deployment experience to Data Scientists and Machine Learning Engineers. It 
+simplifies development by providing a common API for framework and hardware 
+developers to jointly optimize a Deep Learning model; deployment is simplified 
```

Comment:
`jointly` seems slightly off... kinda makes it sound like framework/hardware work is tightly coupled, which is not what we're trying to say

Reply:
Agree and removed .. this phrase might have been a remnant of the blog when we were going to publish  that, which was focused more on the collaboration as a joint effort